app/editor/src/features/admin/folders/FolderFormCollect.tsx

```diff
@@ -161,36 +162,43 @@ export const FolderFormCollect: React.FC = () => {
                 label="Monday"
                 name={`events.0.runOnWeekDays`}
                 value={ScheduleWeekDayName.Monday}
+                onBeforeChange={(value) => selectWeekDays(value)}
```

Comment:
Added callback to clean up the selected values and remove `NA` values when there are other values selected.